.gitignore

```diff
@@ -1,11 +1,3 @@
-.buildlog
```

Comment:
can you revert the changes to this file?

Reply:
No. 😄

Reply:
I really want our repos to show best practices. There is no need to ignore any of those other files/dirs

Reply:
cool @devoncarew ?